lib/src/cosmos_db_containers.dart

```diff
@@ -103,9 +103,12 @@ class CosmosDbContainers {
     try {
       return await open(name);
     } on NotFoundException {
+      if(partitionKey == null){
+        throw "container '$name' not found - can not create with a null partion key";
```

Comment:
Hello, thanks for your PR I'll be happy to accept it but would like first that the code raises an `ApplicationException` instead of a string. `ApplicationException`s are used when app developers fail to provide valid input. Can you update your submission so I can merge it?

Reply:
Will do.

Reply:
done.